content/en/docs/advanced-tasks/fail-forward-upgrades.md

```diff
@@ -0,0 +1,119 @@
+---
+title: "Fail forward upgrades"
```

Comment:
How do we feel about "Enabling fail forward upgrades"?

Reply:
Hmm - looking at the phrasing for the rest of the advanced-tasks changing this phrases would make it less consistent.

Reply:
What about "Opting into fail forward upgrades"? That would remain consistent with the other documents + this one.

Reply:
It probably also makes sense to either remove the description or update it to not be the same as the title 🤣